lib/next_ls/runtime.ex

```diff
@@ -227,6 +223,17 @@ defmodule NextLS.Runtime do
     {:noreply, %{state | compiler_ref: nil}}
   end
 
+  def handle_info({:DOWN, _, :port, port, reason}, %{port: port} = state) do
+    error = {:port_down, reason}
+    state.on_initialized.({:error, error})
```

Comment:
i think that the port could go down after the runtime has initialized, so calling that function here seems not intuitive.

if you say "when not is_map_key(state, :node)" as a clause that implies the node hasn't connected yet

or else it might be better to add another callback to pass in maybe